addons/event_sale/models/event.py

```diff
@@ -6,6 +6,35 @@
 from openerp.osv import fields as old_fields
 
 
+class event_type(models.Model):
+    _inherit = 'event.type'
+    _inherits = {"product.product": "product_id"}
+
+    product_id = fields.Many2one('product.product', 'Product', ondelete='cascade', required=True, domain=[('event_ok', '=', True), ('type' '=' 'service')])
```

Comment:
@tde-banana-odoo ,
Yes, If create an event type "Seminat" then create an automatic product of event type, that the all seminars will be based on the same product.